queries/plugins/text_to_num.py

```diff
@@ -0,0 +1,573 @@
+
+TöluðTalaOgUndirTrilljón →
+    "og" TöluðTalaBilljarðar
+    | TöluðTalaOgUndirBilljarði
+
+TöluðTalaBilljarðar →
+    TöluðTalaUndirBilljarði? TöluðTalaBilljarður
+    | TöluðTalaUndirMilljón? TöluðTalaMilljón TöluðTalaMilljarður
+
+# BILLJÓNIR
+
+TöluðTalaUndirBilljarði →
+    TöluðTalaBilljónir TöluðTalaUndirBilljón?
+    | TöluðTalaBilljónir TöluðTalaOgUndirBilljón
+    | TöluðTalaUndirBilljón
+
+TöluðTalaOgUndirBilljarði →
+    "og" TöluðTalaBilljónir
+    | TöluðTalaOgUndirBilljón
+
+TöluðTalaBilljónir →
+    TöluðTalaUndirBilljón? TöluðTalaBilljón
+    | TöluðTalaUndirMilljón? TöluðTalaMilljón TöluðTalaMilljón
+
+# MILLJARÐAR
+
+TöluðTalaUndirBilljón →
+    TöluðTalaMilljarðar TöluðTalaUndirMilljarði?
+    | TöluðTalaMilljarðar TöluðTalaOgUndirMilljarði
+    | TöluðTalaUndirMilljarði
+
+TöluðTalaOgUndirBilljón →
+    "og" TöluðTalaMilljarðar
+    | TöluðTalaOgUndirMilljarði
+
+TöluðTalaMilljarðar →
+    TöluðTalaUndirMilljarði? TöluðTalaMilljarður
+
+# MILLJÓNIR
+
+TöluðTalaUndirMilljarði →
+    TöluðTalaMilljónir TöluðTalaUndirMilljón?
+    | TöluðTalaMilljónir TöluðTalaOgUndirMilljón
+    | TöluðTalaUndirMilljón
+
+TöluðTalaOgUndirMilljarði →
+    "og" TöluðTalaMilljónir
+    | TöluðTalaOgUndirMilljón
+
+TöluðTalaMilljónir →
+    TöluðTalaUndirMilljón? TöluðTalaMilljón
+
+# ÞÚSUND
+
+TöluðTalaUndirMilljón →
+    TöluðTalaÞúsundir TöluðTalaUndirÞúsund?
+    | TöluðTalaÞúsundir TöluðTalaOgUndirÞúsund
+    | TöluðTala10Til19Hundruð TöluðTalaUndirHundrað?
+    | TöluðTala10Til19Hundruð TöluðTalaOgUndirHundrað?
+    | TöluðTalaUndirÞúsund
+
+TöluðTalaOgUndirMilljón →
+    "og" TöluðTalaÞúsundir
+    | TöluðTalaOgUndirÞúsund
+
+TöluðTalaÞúsundir →
+    TöluðTalaUndirÞúsund? TöluðTalaÞúsund
+
+TöluðTala10Til19Hundruð →
+    TöluðTala10Til19 TöluðTalaHundrað
+
+# HUNDRUÐ
+
+TöluðTalaUndirÞúsund →
+    TöluðTalaHundruð TöluðTalaTugurOgEining?
+    | TöluðTalaHundruð TöluðTalaOgUndirHundrað?
+    | TöluðTalaUndirHundrað
+
+TöluðTalaOgUndirÞúsund →
+    "og" TöluðTalaHundruð
+    | TöluðTalaOgUndirHundrað
+
+TöluðTalaHundruð →
+    TöluðTala1Til9? TöluðTalaHundrað
+
+# UNDIR HUNDRAÐ
+
+TöluðTalaUndirHundrað →
+    TöluðTalaTugurOgEining
+    | TöluðTalaTugir
+    | TöluðTala10Til19
+    | TöluðTala1Til9
+
+TöluðTalaOgUndirHundrað →
+    "og" TöluðTalaTugir
+    | "og" TöluðTala10Til19
+    | "og" TöluðTala1Til9
+
+TöluðTalaTugurOgEining →
+    TöluðTalaTugir "og" TöluðTala1Til9
+
+###
+
+TöluðTala0 → 'núll'
+
+TöluðTala1Til9 →
+    'einn:to'
+    | 'tveir:to'
+    | 'þrír:to'
+    | 'fjórir:to'
+    | "fimm"
+    | "sex"
+    | "sjö"
+    | "átta"
+    | "níu"
+
+TöluðTala10Til19 →
+    "tíu"
+    | "ellefu"
+    | "tólf"
+    | "þrettán"
+    | "fjórtán"
+    | "fimmtán"
+    | "sextán"
+    | "sautján"
+    | "átján"
+    | "nítján"
+
+TöluðTalaTugir →
+    "tuttugu"
+    | "þrjátíu"
+    | "fjörutíu"
+    | "fimmtíu"
+    | "sextíu"
+    | "sjötíu"
+    | "áttatíu"
+    | "níutíu"
+
+TöluðTalaHundrað → 'hundrað'
```

Comment:
Missing :cat (word category specifier)? Also be careful that in plural, there are both 'hundruð' and 'hundruðir' (and 'þúsund' and 'þúsundir').

Reply:
Added category specifiers! We may need to add hundruðir as an entry to ord.add.csv - as it is relatively common (but wrong, according to BÍN). Added þúsundir:kvk for plural.